src/pyOpenMS/pxds/AASequence.pxd

```diff
@@ -136,7 +137,7 @@ cdef extern from "<OpenMS/CHEMISTRY/AASequence.h>" namespace "OpenMS":
 cdef extern from "<OpenMS/CHEMISTRY/AASequence.h>" namespace "OpenMS::AASequence":
         
         # static members
-        AASequence fromString(String s, bool permissive) except + nogil   # wrap-attach:AASequence wrap-as:fromStringPermissive
+        AASequence fromString(String s, bool permissive) except + nogil   # wrap-attach:AASequence wrap-as:fromStringPermissive wrap-doc:deprecated. Use AASequence(String) instead.
```

Comment:
I think we then also need to wrap the (String) version.  I think default parameter are not supported in pxd files.

Reply:
I think I did that here? https://github.com/OpenMS/OpenMS/blob/688a53d1817fe23d65edfae15957efbbd3eee6e7/src/pyOpenMS/pxds/AASequence.pxd#L24

Reply:
Ah right  I did not see the previous change